CommandLine/libEGM/CMakeLists.txt

```diff
@@ -1,68 +1,37 @@
 cmake_minimum_required(VERSION 3.11)
```

Comment:
Create an RGM branch, pull this branch, send a pr to RGM first before ever merging this here. That way we'll know RGM is still working.